webapp/packages/plugin-data-viewer/src/ValuePanelPresentation/TextValue/TextValuePresentation.m.css

```diff
@@ -0,0 +1,6 @@
+.toolsContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+}
```

Comment:
this can be replaced with `<Container/>` component

Reply:
`<Container/>` breaks all of the layout for **ValuePanel**. I tend to think this should be done during reshadow refactoring cause now it cannot be implemented easy peasy way